installer/vm_providers/_snap.py

```diff
@@ -0,0 +1,329 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
```

Comment:
Is it possible we do not need this file?

Reply:
I'm pretty sure this is what's doing the `snap install microk8s --classic`, but I can make a test build